src/cli-validator/runValidator.js

```diff
@@ -236,8 +237,9 @@ const processInput = async function(program) {
       results = validator(swagger, configObject);
     } catch (err) {
       printError(chalk, 'There was a problem with a validator.', getError(err));
-      // Uncomment the line below to see the stack trace when the validator fails
-      // console.log(err.stack);
+      if (debug) {
+        console.log(err.stack);
+      }
```

Comment:
Since you are adding this, will you make the same change to the block above, [line 224](https://github.com/IBM/openapi-validator/pull/151/files#diff-55deb9d411eefb9ce224c9daec223bc1R224)?

Reply:
Fixed!